errors/middleware-upgrade-guide.md

```diff
@@ -9,6 +9,8 @@ This upgrade guide will help you understand the changes and how to migrate your
 
 You can start upgrading your Middleware usage today with the latest canary release (`npm i next@canary`).
 
+If you have ESLint configured, you will need to run `npm i eslint-config-next@canary --force` to upgrade your ESLint configuration to the latest canary release. You might also need to restart VSCode for the changes to take effect.
```

Comment:
```suggestion
If you have ESLint configured, you will need to run `npm i eslint-config-next@canary --save-dev` to upgrade your ESLint configuration to the latest canary release. You might also need to restart VSCode for the changes to take effect.
```
I don't think the force flag needs to be recommended here, did you run into issues without it? We do recommend this be a devDependency though

Reply:
Yeah I ran into this issue without using `--force`:

![CleanShot 2022-06-24 at 22 47 14](https://user-images.githubusercontent.com/28986134/175757323-441e3de6-207d-4626-a59a-75bbf9d7dcbd.png)

Reply:
VSCode told me to use `--force` and that resolved the issue

Reply:
Hmm I think we can remove the `peerDependency` to resolve that as it's technically not needed

Reply:
Opened PR to fix this here https://github.com/vercel/next.js/pull/38009

Reply:
I also don't think this needs to mention `canary`? Maybe say that the version should match the `next` version?